lib/sandbox.nix

```diff
@@ -65,40 +108,5 @@ in
     fi
 
     set -x
-    ( exec ${bubblewrap}/bin/bwrap \
-        --dir "$HOME" \
-        --proc /proc \
-        --dev /dev \
-        --dev-bind-try /dev/snd /dev/snd \
-        --dev-bind-try /dev/video0 /dev/video0 \
-        --dev-bind-try /dev/video1 /dev/video1 \
-        --dev-bind /dev/dri /dev/dri \
-        --ro-bind /nix /nix \
-        --ro-bind /etc /etc \
-        --ro-bind-try /bin /bin \
-        --ro-bind-try /usr/bin/env /usr/bin/env \
-        --bind-try "$HOME/.cache/fontconfig" "$HOME/.cache/fontconfig" \
-        --tmpfs /run \
-        --tmpfs /tmp \
-        --ro-bind "$SHELL" "$SHELL" \
-        --ro-bind-try "$HOME/.config/zsh/.zshenv" "$HOME/.config/zsh/.zshenv" \
-        --ro-bind-try "$HOME/.config/zsh/.zshrc" "$HOME/.config/zsh/.zshrc" \
-        --ro-bind-try "$HOME/.config/zsh/plugins" "$HOME/.config/zsh/plugins" \
-        --ro-bind-try "$HOME/.zshenv" "$HOME/.zshenv" \
-        --setenv DISPLAY ":0" \
-        --ro-bind /tmp/.X11-unix/X0 /tmp/.X11-unix/X0 \
-        --ro-bind "$XAUTHORITY" "$XAUTHORITY" \
-        --new-session \
-        --die-with-parent \
-        --unshare-all \
-        --setenv PATH ${lib.makeBinPath [
-      coreutils
-      bashInteractive
-    ]} \
-        ${quoteShellArgs extraBubblewrapOptions} \
-        ${emacsDirectoryOpts} \
-        --ro-bind ${initEl} ${userEmacsDirectory'}/init.el \
-        $opts \
-        ${emacs}/bin/emacs ${quoteShellArgs emacsArguments} "$@"
-      )
+    ${command}
```

Comment:
It is common for a bwrap script like this to enclose the command line with `( exec ... )`. See https://github.com/valoq/bwscripts/tree/master/profiles for example.